pipeline-yaml_5b5744d40428631d7a8940a9.md

```diff
@@ -371,10 +372,59 @@ Error: "There are at least two blocks with same name: Build Go project"
 Semaphore assigns its own unique names to nameless `blocks` items, which are
 displayed in the Semaphore 2.0 UI.
 
+### dependencies in blocks
+
+The `dependencies` property of a block is used to define the flow of execution
+between blocks within a pipeline.
```

Comment:
Prefer active over passive voice. This applies to all writing.

Reply:
It seems like I might want to use this when running blocks in parallel, but it's not clear. Can you rephrase?

Assume that the reader is trying to figure out if this is relevant to them from the first paragraph.

Reply:
There's almost always a way to say the same thing in active voice, which sounds more friendly, without losing the technical rigor.

For example:

_When your pipeline is running blocks in parallel, you can use the `dependencies` property to define the flow of execution for subsequent blocks._